theme/static/css/dist/styles.css

```diff
@@ -634,17 +634,17 @@ select {
 
 [type='checkbox'],[type='radio'] {
   -webkit-appearance: none;
-     -moz-appearance: none;
-          appearance: none;
+  -moz-appearance: none;
+  appearance: none;
   padding: 0;
   -webkit-print-color-adjust: exact;
           print-color-adjust: exact;
   display: inline-block;
   vertical-align: middle;
   background-origin: border-box;
   -webkit-user-select: none;
-     -moz-user-select: none;
-          user-select: none;
```

Comment:
I get it, visual indent looks clean, but if this should be used for css files, I need a linting tool that can enforce it.
Only removed the sections that editorconfig-check complained about.